llvm/lib/Transforms/IPO/LowerTypeTests.cpp

```diff
@@ -1227,6 +1239,18 @@ void LowerTypeTestsModule::lowerTypeTestCalls(
       ++NumTypeTestCallsLowered;
       Value *Lowered = lowerTypeTestCall(TypeId, CI, TIL);
       if (Lowered) {
+        if (TotalCallCount) {
+          auto *CIF = CI->getFunction();
+          if (auto EC = CIF->getEntryCount())
```

Comment:
This if should also use braces. It would also be good to combine it with the next line into a single if condition to reduce nesting.